client/lib/purchases/index.ts

```diff
@@ -419,6 +419,29 @@ export function isCancelable( purchase: Purchase ) {
 	return purchase.canDisableAutoRenew;
 }
 
+/**
+ * Similar to isCancelable, but doesn't rely on the purchase's cancelability
+ * Checks if auto-renew is enabled for purchase, returns true if auto-renew is ON
+ * Returns false if purchase is included in plan, purchases included with a plan can't be cancelled
+ * Returns false if purchase is expired
+ */
+
+export function canAutoRenewBeTurnedOff( purchase: Purchase ) {
+	if ( isIncludedWithPlan( purchase ) ) {
+		return false;
+	}
+
+	if ( isExpired( purchase ) ) {
+		return false;
+	}
+
+	if ( hasAmountAvailableToRefund( purchase ) ) {
```

Comment:
Running through the manual tests again, I ran into an 'issue' I guess:

![image](https://user-images.githubusercontent.com/16580129/211123032-bb620797-e7ef-4f24-bf4e-7c97e76916fc.png)

In situations where a refund is available, `canAutoRenewBeTurnedOff` will return true, which will then show the Cancel button.

This is expected, but if you were to manually turn OFF autorenew then the Cancel button would still show due to this condition returning true. I am not sure if this is really an issue. Essentially, the Cancel button would only show until the purchase was no longer refundable. Once enough time passes and `Store_Subscription::is_refundable()` returns `false` then the Cancel button would be replaced by Remove.

So I think this behavior is expected, thoughts? @sirbrillig 

![image](https://user-images.githubusercontent.com/16580129/211124901-3421073d-e0b2-45d0-8979-d95e0805afcb.png)

Reply:
This doesn't appear to be an issue. From testing, it is expected that _if_ there is an available refund _then_ the cancel button should show regardless of the auto-renew(able) state. This is to ensure a customer can self-serve their refund when eligible.